bundles/org.openhab.binding.tradfri/README.md

```diff
@@ -65,17 +78,33 @@ The control outlet supports the `power` channel.
 
 A blind or curtain supports, beside `battery_level` and `battery_low` channels,  a `positon` channel.
 
-Refer to the matrix above.
-
-| Channel Type ID   | Item Type     | Description                                            |
-|-------------------|---------------|--------------------------------------------------------|
-| brightness        | Dimmer        | The brightness of the bulb in percent                  |
-| color_temperature | Dimmer        | color temperature from 0% = cold to 100% = warm        |
-| color             | Color         | full color                                             |
-| battery_level     | Number        | battery level (in %)                                   |
-| battery_low       | Switch        | battery low warning (<=10% = ON, >10% = OFF)           |
-| power             | Switch        | power switch                                           |
-| position          | Rollershutter | position of the blinds from 0% = open to 100% = closed |
+An air purifier supports:
+* a `fan_mode` and `fan_speed` channels, which allows for control of the fan and reading of the current speed.
```

Comment:
Here and below, "a " should probably be removed the they are now mentioned as "channels" (plural):
```suggestion
* `fan_mode` and `fan_speed` channels, which allows for control of the fan and reading of the current speed.
```

Reply:
very valid remarks, fixed